quisby.py

```diff
@@ -459,12 +459,20 @@ def compare_data(s_list, comp_list, noti_flag, exclude):
     noti_flag = True
     if args.no_notify:
         noti_flag = False
+    
+    default_config_path = os.path.expanduser("~/.quisby/config/config.ini")
 
-    if not args.config:
-        custom_logger.error("No configuration path mentioned.")
-        exit(1)
-    else:
+    if args.config:
         util.config_location = args.config
```

Comment:
This should be replaced by the `default` parameter in `add_argument`
https://docs.python.org/3/library/argparse.html#default

Should look roughly like `parser.add_argument("--config", default="~/.quisby/config/config.ini", <REST OF PARAMS>)`